src/fed_dbm.cpp

```diff
@@ -586,10 +588,10 @@ namespace dbm
 
         cindex_t dim = pdim();
         raw_t* mdbm = getCopy();
-        uint32_t touched[bits2intsize(dim)];
+        std::vector<uint32_t> touched = std::vector<uint32_t>(bits2intsize(dim));
```

Comment:
use `auto`. Actually I prefer AAA, it's just here you keep repeating the type name :-)

Reply:
it would be cool if `std::vector<bool>` worked here